File: go/slot_fee.go
// this file is @generated
package meteroid

type SlotFee struct {
	InitialSlots int32 `json:"initial_slots"`

	MaxSlots *int32 `json:"max_slots,omitempty"`

	MinSlots *int32 `json:"min_slots,omitempty"`

	Unit string `json:"unit"`

	UnitRate string `json:"unit_rate"`
}

File: python/meteroid/models/slot_fee.py
# this file is @generated
import dataclasses
import typing as t
from decimal import Decimal

from ..serialization import BaseModel


@dataclasses.dataclass
class SlotFee(BaseModel):
    initial_slots: int

    unit: str

    unit_rate: Decimal

    max_slots: t.Optional[int] = None

    min_slots: t.Optional[int] = None
